### DISCUSSION TITLE
Troubleshooting Akvorado. Running the command curl -s http://127.0.0.1:8080/api/v0/inlet/flows\?limit=1 just hangs

### DISCUSSION THREAD

**Discussion by honeyfairy** [akvorado/akvorado#1092] [Q&A]
I'm trying to troubleshoot my Akvorado installation. I have tried it on multiple containers, VMs in my proxmomx enviroment. The demo data shows up, but when I remove the demo data and try to send my flows nothing is received.
I not set up Akvorado on Vultr Docker VM ( https://www.vultr.com/marketplace/apps/docker/?_gl=1*12x9tcm*_ga*Mjk4MDMyNjE5LjE3MDc1NTk1NTc.*_ga_K6536FHN4D*MTcwODE4NDAyOC41LjEuMTcwODE4NTQ5MS4yOS4wLjA.)

Still it does not seem to work. 
So, I visited the troubleshooting page, and it tells me to run curl -s http://127.0.0.1:8080/api/v0/inlet/flows\?limit=1 
But this command just hangs, and the page does not tell me what to do if it hangs

**Comment by vincentbernat** [comment]
It means you are not receiving any flows or their processing is failing. Continue the troubleshooting guide.
